components/brave_today/browser/channels_controller.h

```diff
@@ -32,19 +32,10 @@ class ChannelsController {
 
   static Channels GetChannelsFromPublishers(const Publishers& publishers,
                                             PrefService* prefs);
-
-  // Get all the Locales the user has subscribed to channels in.
-  std::vector<std::string> GetChannelLocales() const;
-
-  // Gets all the locales the user has subscribed to a channel in.
-  std::vector<std::string> GetChannelLocales(
-      const std::string& channel_id) const;
   void GetAllChannels(ChannelsCallback callback);
   mojom::ChannelPtr SetChannelSubscribed(const std::string& locale,
```

Comment:
I find it slightly odd how for some methods we have to go through the `pref_utils` (ie. `GetChannelLocales` & `GetChannelSubscribed`) but for others we need to go through the `ChannelsController`. I see the logic in this split (`SetChannelSubscribed` does other stuff), but it does feel slightly weird.

Reply:
Yeah I think I'm with you on that now, with the way we have things split currently. I was looking at it more like abstracting the prefs stuff, but really the priority here is just for it to be testable. I'm going to change it to be static methods on the existing class.